.github/workflows/compress.yml

```diff
@@ -35,6 +35,6 @@ jobs:
           # The default hash digest is set to 20 chars in Webpack
           strip-hash: "\\.(\\w{20})\\.js$"
 
-          # reporting on “legacy” and “variant” bundles muddies the water
+          # report exclusively on the “web” bundle – no “apps”, “variant” or other
           # https://github.com/preactjs/compressed-size-action#customizing-the-list-of-files
-          pattern: "dotcom-rendering/dist/*.web.*.js"
+          pattern: 'dotcom-rendering/dist/*.web.????????????????????.js'
```

Comment:
I think this (or a similar) comment from L35 could be helpful here as well: `# The default hash digest is set to 20 chars in Webpack`

Reply:
```suggestion
          # The default hash digest is set to 20 chars in Webpack
          pattern: 'dotcom-rendering/dist/*.web.????????????????????.js'
```